servers/push-server/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { Worker, isMainThread } from 'worker_threads';
+import Sentry from './sentry';
+import { worker } from './worker';
+import { numWorkers, msBetweenStarts } from './config';
+
+if (isMainThread) {
```

Comment:
we could replace all this worker logic with https://github.com/bbc/sqs-consumer and really clean up the logic